### PR TITLE
Documentation - Link & Instructions 

### PR DESCRIPTION
**Pull request by cgrant**
Updated a link on the main readme to point back to the correct link (https://cloud.google.com/kubernetes-engine/docs/how-to/kubemci)
Added instructions for installing kubemci itself in the example readme.

### CODE REVIEW

**Comment by G-Harmon** [comment]
thanks for the PR! merging...

**Comment by G-Harmon** [comment]
actually, can you please squash your 2 commits please?

**Comment by cgrant** [comment]
Squashed :)

File: examples/zone-printer/README.md [hunk 1]
```diff
@@ -56,6 +56,22 @@ used by `kubemci` to provision the multi-cluster Ingress.
     gcloud components install kubectl
     ```
 
+4. Install **`kubemci`**:
+
+    You can install `kubemci` by running:
+
+    ```shell
+    wget https://storage.googleapis.com/kubemci-release/release/latest/bin/linux/amd64/kubemci
```
Comment (nikhiljindal) [inline examples/zone-printer/README.md]: Note that we have 2 binaries:
Linux, amd64: https://storage.googleapis.com/kubemci-release/release/latest/bin/linux/amd64/kubemci
darwin (MacOSX), amd64: https://storage.googleapis.com/kubemci-release/release/latest/bin/darwin/amd64/kubemci

You could also point to https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/releases if users want a specific version

For reference, look at "Download the kubemci command-line tool" section in https://cloud.google.com/kubernetes-engine/docs/how-to/kubemci

File: examples/zone-printer/README.md [hunk 2]
```diff
@@ -56,6 +56,22 @@ used by `kubemci` to provision the multi-cluster Ingress.
...
+    chmod +x ./kubemci
+
```
Comment (nikhiljindal) [inline examples/zone-printer/README.md]: nit: can remove extra blank line

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ kubemci is a tool to configure Kubernetes ingress to load balance traffic across
 multiple Kubernetes clusters.
 
 This is a Google Cloud Platform [beta](https://cloud.google.com/terms/launch-stages) tool, suitable for limited production use cases:
-https://cloud.google.com/kubernetes-engine/docs/how-to/setup-multi-cluster-ingress
```
Comment (nikhiljindal) [inline README.md]: This link works again. Please revert the change.

Comment (cgrant) [reply README.md]: Reverted

File: examples/zone-printer/README.md (hunk 1 above)
Comment (cgrant) [reply examples/zone-printer/README.md]: updated

File: examples/zone-printer/README.md (hunk 2 above)
Comment (cgrant) [reply examples/zone-printer/README.md]: removed extra line

**Comment by nikhiljindal** [comment]
/lgtm.
The test failure is unrelated
